src/routes/disclaimer/containers/__tests__/DisclaimerPage.spec.js

```diff
@@ -63,15 +62,14 @@ describe('DisclaimerPage', () => {
 
   describe('connect', () => {
     test('should map state to props', () => {
-      const store = new Store()
-      store.init({
+      const store = mockStore({
```

Comment:
🔥 You need a real implementation of a Store in this test. Merge conflicts incoming!

Reply:
This was part of the merge conflict 🤔

Reply:
See above. Maybe @klinzo knows more. But I'm pretty sure this will fail.

Reply:
yep